pyvrp/cpp/Solution.h

```diff
@@ -148,52 +148,50 @@ class Individual
      */
     [[nodiscard]] Duration timeWarp() const;
 
-    bool operator==(Individual const &other) const;
+    bool operator==(Solution const &other) const;
 
-    Individual &operator=(Individual const &other) = delete;  // is immutable
-    Individual &operator=(Individual &&other) = delete;       // is immutable
+    Solution &operator=(Solution const &other) = delete;  // is immutable
+    Solution &operator=(Solution &&other) = delete;       // is immutable
 
-    Individual(Individual const &other) = default;
-    Individual(Individual &&other) = default;
+    Solution(Solution const &other) = default;
+    Solution(Solution &&other) = default;
 
     /**
-     * Constructs a random individual using the given random number generator.
+     * Constructs a random solution using the given random number generator.
      *
-     * @param data           Data instance describing the problem that's being
-     *                       solved.
-     * @param rng            Random number generator.
+     * @param data Data instance describing the problem that's being solved.
+     * @param rng  Random number generator.
      */
-    Individual(ProblemData const &data, XorShift128 &rng);
+    Solution(ProblemData const &data, XorShift128 &rng);
 
     /**
-     * Constructs an individual having the given routes as its solution.
+     * Constructs an solution with the given routes.
```

Comment:
Here are some other matches:
```
pyvrp/diversity/tests/test_broken_pairs_distance.py
16:    # BPD of an solution and itself should be zero.

pyvrp/Population.py
119:        # Note: the CostEvaluator is required here since adding an solution
```